datadog_checks_base/datadog_checks/base/utils/yaml.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import logging
+
+import yaml
+
+from ..ddyaml import pyyaml_load, pyyaml_load_all
+
+log = logging.getLogger(__name__)
+
+
+def yaml_load_force_loader(stream, Loader):
```

Comment:
Should this method live in `..ddyaml` instead?

Reply:
Since it's used for a really specific use case I think it's better to have it as an util function, seperated from the monkeypatching, but no real strong opinion on that

Reply:
If we don't anticipate other integrations needing it and we want to make it really specific we can always put this logic into XenServer itself.
I'm in favor of doing one or the other depending, but less in favor of creating another `yaml` utils in datadog_checks_base

Reply:
I prefer to have these function in the base package, it's specific but probably not unique to xenserver.
I've moved the functions to `ddyaml`, but they must be explicitly exported in the `__init__.py` i believe, so I don't know if it's better than having them in a separate utils file